### ISSUE TITLE
Remove `?` from hash

### ISSUE THREAD

**Issue by MattMS** [ramda/repl#39]
Unless there is a reason that the `?` is required, then it seems like parsing the code from the hash would be simpler if it didn't require removing the `?` each time.

**Comment by craigdallimore** [comment]
Agreed - at present the "try in repl" part of the ramda docs includes the `?` in its links, so we probably need to account it until we fix it on that side.

**Comment by MattMS** [comment]
There [was a merged PR](https://github.com/ramda/ramda.github.io/pull/145) that changed the `?` to a `;` in preparation for this.
I just checked the docs page and it appears to be working.

So you should be able to include the first character and it would still be valid JS.
Then we can update the button click handler to exclude that character.
